projects/ngx-feature-flag/src/lib/feature-wrapper/feature-toggle.component.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpClientModule } from '@angular/common/http';
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { of } from 'rxjs';
+import { Feature } from '../models/feature';
+import { FeatureTogglesService } from '../service/feature-toggles.service';
+import { FeatureToggleComponent } from './feature-toggle.component';
+
+describe('FeatureToggleComponent', () => {
+  let featureServiceMock: jasmine.SpyObj<FeatureTogglesService>;
+  let hostFixture: ComponentFixture<TestHostComponent>;
+
+  const feature: Feature = {
+    attributes: {
+      name: 'Planner',
+      description: 'Description',
+      enabled: true,
+      'enablement-level': 'beta',
+      'user-enabled': true
+    },
+    id: 'Planner'
+  };
+
+  @Component({
+    selector: `f8-host-component`,
+    template: `<f8-feature-toggle featureName="Planner"><div user-level>My content here</div></f8-feature-toggle>`
```

Comment:
This template needs to be (re- ?)updated for the new `ng-template` syntax used by the `f8-feature-toggle` component